sdk/keyvault/azsecrets/models.go

```diff
@@ -52,49 +39,79 @@ type Secret struct {
 	// The secret management attributes.
 	Properties *Properties `json:"attributes,omitempty"`
 
-	// The content type of the secret.
-	ContentType *string `json:"contentType,omitempty"`
-
 	// The secret id.
 	ID *string `json:"id,omitempty"`
 
-	// Application specific metadata in the form of key-value pairs.
-	Tags map[string]string `json:"tags,omitempty"`
+	// The name of the secret
+	Name *string
```

Comment:
If there was an error with the request this will be nil, like all other properties. It technically doesn't have to be a pointer but I made it a pointer for consistency reasons